packages/clerk-js/webpack.config.js

```diff
@@ -170,22 +170,58 @@ const typescriptLoaderDev = () => {
   };
 };
 
-/** @type { () => (import('webpack').RuleSetRule) } */
+/**
+ * Used in outputs that utilize chunking, and returns a URL to the stylesheet.
+ * @type { () => (import('webpack').RuleSetRule) }
+ */
 const clerkUICSSLoader = () => {
-  // This emits a module exporting the URL to the styles.css file.
+  // This emits a module exporting a URL to the styles.css file.
   return {
     test: /packages\/ui\/dist\/styles\.css/,
     type: 'asset/resource',
   };
 };
 
-/** @type { () => (import('webpack').Configuration) } */
-const commonForProd = () => {
+/**
+ * Used in outputs that _do not_ utilize chunking, and returns the contents of the stylesheet.
+ * @type { () => (import('webpack').RuleSetRule) }
+ */
+const clerkUICSSSourceLoader = () => {
+  // This emits a module exporting the contents of the styles.css file.
+  return {
+    test: /packages\/ui\/dist\/styles\.css/,
+    type: 'asset/source',
+  };
+};
+
+/**
+ * Used for production builds that have dynamicly loaded chunks.
+ * @type { () => (import('webpack').Configuration) }
+ * */
+const commonForProdChunked = () => {
   return {
-    devtool: undefined,
     module: {
       rules: [svgLoader(), typescriptLoaderProd(), clerkUICSSLoader()],
     },
+  };
+};
+
+/**
+ * Used for production builds that combine all files into one single file (such as for Chrome Extensions).
+ * @type { () => (import('webpack').Configuration) }
+ * */
+const commonForProdBundled = () => {
```

Comment:
We could also do something like `commonForProd({ bundled: true })` and do everything within the `commonForProd` function.